src/main/scala/consul/v1/common/ConsulRequestBasics.scala

```diff
@@ -1,13 +1,16 @@
 package consul.v1.common
 
+import com.ning.http.client.AsyncHttpClientConfig
 import consul.v1.common.Types.DatacenterId
 import play.api.libs.json._
-import play.api.libs.ws.{WS, WSRequest, WSResponse}
-import play.api.Play.current
+import play.api.libs.ws.{WS, WSClient, WSRequest, WSResponse}
+import play.api.libs.ws.ning.NingWSClient
+import play.api.Application
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.{Failure, Success, Try}
 
-class ConsulRequestBasics(token: Option[String]) {
+class ConsulRequestBasics(token: Option[String], client: WSClient) {
+  def this(token: Option[String])(implicit app: Application) = this(token, WS.client)
```

Comment:
hey travis! you are absolutely right. a fix is on the way! thank you for pointing it out